src/components/Hero.astro

```diff
@@ -5,33 +5,42 @@ import cnpicon from '../images/cnp-icon.svg';
 const { heroText, heroImage, ctaText, ctaLink } = Astro.props;
 ---
 
+<div class="background">
+  <img src={heroImage} alt="decorative" class="full-img" />
+  <div class="content">
+    <PortableText value={heroText} />
+  </div>
+  {
```

Comment:
As per meeting discussion can you move this in the "content" block and style it to look like the CTA?

Reply:
done